### DISCUSSION TITLE
Slow load times

### DISCUSSION THREAD

**Discussion by andyburgessmd** [Budibase/budibase#11832] [Help]
Can anything be done about the slow loading times? 
I have an application where it takes 10 seconds to load each page and its unusable. Its not doing much but developer tools shows me the requests that are taking up the time.
- A request to https://mytenant.budibase.app/api/self takes 2.95s to get a response from the server, and the contents is about me and the budibase account (billing, quotas, applications) most of which is not needed in the application
- Next is a request to https://mytenant.budibase.app/api/ta_users/myuserid/enrich and takes 4.44s to get a response from the server and contains a very similar set of information as the first

From the total load time of 10.06s these two requests are taking 7.39s (73%) of the load time.

Am I doing something wrong? Does anyone else get this?

Hosting: CLOUD

**Comment by stellarfinn** [comment]
I can only provide my anecdotal experience but it sounds similar. I've found that if you're using formulas in your database (dynamic in particlar) each formula field slows down the load time (I assume to the db having to run all those formulas). If you are using formulas in your database and specifically the dynamic one, that is probably the issue.

**Reply by andyburgessmd** [reply]
Thanks @stellarfinn. I saw the other discussion about that and although I am using formulas they are all static, I did switch some from dynamic but it did not make much difference in the load time, maybe in comparison to the two requests above.
